src/gui/optionsdlg.ui

```diff
@@ -122,8 +122,8 @@
             <rect>
              <x>0</x>
              <y>0</y>
-             <width>501</width>
-             <height>893</height>
+             <width>560</width>
```

Comment:
Are all of these size changes required?

Reply:
Qt creator in design mode did them automatically when it saved the file, any idea how to stop it?

Reply:
No, sorry. I use standalone Designer app, and it does not resize forms on opening.

Reply:
I'll try that.

Reply:
Still the same.

Reply:
It may happen when your visual style requires widgets to be bigger than that used be the previous author and designer app has to expand the form to accommodate expanded widgets.

Reply:
Yeah probably, i reset my system font sizes to default and although it still changes these values, they are closer to the original.